dev/source/docs/building-setup-linux.rst

```diff
@@ -79,14 +111,12 @@ Then reload your PATH by using the "dot" command in a terminal
 ::
 
     . ~/.bashrc
-
-Setup for other Distributions
-=============================
-
+    
```

Comment:
maybe a title here like :
STM toolchain

Reply:
the right toolchain tar is gcc-arm-none-eabi-6-2017-q2-update-linux.tar.bz2 not the 4.9 one (L152) and L157 the name is also wrong.
We should put Compiler section (L144) before Permissions ones (L133)
And put back the CCache install at end

Reply:
please check that I guessed correctly at the right path to the compiler dir...

Reply:
@khancyr Pierre?